Simplenote/src/main/res/values/strings.xml

```diff
@@ -113,10 +113,10 @@
 
     <!-- Tags -->
     <string name="add_tag">Add Tag</string>
-    <string name="dialog_tag_add_button_negative">@android:string/cancel</string>
-    <string name="dialog_tag_add_button_positive">@string/save</string>
-    <string name="dialog_tag_add_hint">@string/tag_name</string>
-    <string name="dialog_tag_add_title">@string/add_tag</string>
+    <string name="dialog_tag_add_button_negative" translatable="false">@android:string/cancel</string>
```

Comment:
Nice catch, I think I'd personally have missed the case for `@android:string` (and would only have thought about searching for `@string/`) 👍

I noticed that's a case that we also forgot about in our Peril rule (which only checks for `@string` not `@android:string`), so probably worth fixing by adding those to the Peril rule too